### ISSUE TITLE
feed test set multiple times in ResNet

### ISSUE THREAD

**Issue by deeplearningmachine** [ppwwyyxx/tensorpack#765]
Is there a way to predict the test data set multiple times and then average the logits in ResNet?

**Comment by ppwwyyxx** [comment]
If you can do something once, then you can do something multiple times. What is the issue?

**Comment by deeplearningmachine** [comment]
any build-in function in Tensorpack?

**Comment by ppwwyyxx** [comment]
No there isn't.
